src/dynamodb/batch/batch-delete.js

```diff
@@ -0,0 +1,62 @@
+const { BatchWriteCommand } = require('@aws-sdk/lib-dynamodb');
```

Comment:
If we're grouping this file under /batch, shouldn't it just be called delete.js then? /batch/batch- seems redundant?

Reply:
Agree, it's just that the naming with batch-write vs batch-delete is not nice and gets even worse if we omit batch from the name... But will change it.